### PR TITLE
Localize generated debugger package.json strings

### PR DESCRIPTION
**Pull request by WardenGnaw**
This PR updates the generateOptionsSchema to create keys for package.nls.json that are used in package.json for localization.

This will automatically take the english strings in OptionsSchema.json + etc and convert them into keys to use for package.nls.json and update package.nls.json.

The keys that are generated with start with `generateOptionsSchema` and that will be used to clean up old keys.

### CODE REVIEW

**Review by dibarbet — APPROVED** [review]
I guess a nice side effect is that updates to the package.json will no longer remove the \u200b characters!

**Comment by gregg-miskelly** [comment]
Can we remove this now? https://github.com/dotnet/vscode-csharp/blob/066d2fc260b1e7c66a2b6d789a5c5d7f3acd7814/test/unitTests/packageJson.test.ts#L12

**Review by gregg-miskelly — APPROVED** [review]
Otherwise looks good to me

File: package.nls.json [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+  "csharp.debug.pipeTransport.linux.pipeEnv": "Environment variables passed to the pipe program.",
+  "csharp.debug.suppressJITOptimizations": "If true, when an optimized module (.dll compiled in the Release configuration) loads in the target process, the debugger will ask the Just-In-Time compiler to generate code with optimizations disabled. [More information](https://aka.ms/VSCode-CS-LaunchJson#suppress-jit-optimizations)",
+  "csharp.debug.symbolOptions": "Options to control how symbols (.pdb files) are found and loaded.",
+  "csharp.debug.symbolOptions.searchPaths": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
+  "csharp.debug.symbolOptions.searchMicrosoftSymbolServer": "If 'true' the Microsoft Symbol server (https\u200b://msdl.microsoft.com\u200b/download/symbols) is added to the symbols search path. If unspecified, this option defaults to 'false'.",
```
[the remark this answers is not in the document]
Comment (WardenGnaw) [reply package.nls.json:61]: Yep. I can do something like this: 
> "comment": [ "Markdown text between \`\` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the \`\`) should not be altered." ] },

See https://github.com/microsoft/vscode-cpptools/blob/44e3bbe77379de0ac2afde2df25dd6597040d240/Extension/package.nls.json#L51C293-L51C496

**Review by gregg-miskelly — APPROVED** [review]
:shipit:

**Pull request merged**